src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -664,6 +710,15 @@ public Metric getTicketMetric(long id) {
             handle(Metric.class, "ticket_metric")));
   }
 
+  public Iterable<Audit> getTicketAudits(Map<String, List<String>> parameters) {
```

Comment:
I'm under the impression this adds the ability to push additional parameters to the endpoint. 
While I understand the idea (not wait for the library to be updated if a parameter is added to the library, I'm not sure this is desirable, since it goes against what was done up until now in the library. At the very least this should be split into its separate PR for further discussion.